style.css

```diff
@@ -445,3 +451,387 @@ h4 {
+    grid-template-rows: 1fr;
+    width: 80vw;
+    margin-top: 40%;
+    gap: 150px;
+    justify-items: stretch;
+    padding-left: 10%;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+
+  .page-3-desktop p {
+    width: 90%;
+  }
+
+  .but {
+    width: 120px;
+  }
+
+  .alltext1 ul {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px;
+    padding-left: 0;
+  }
+
+  .second-grid1 {
+    display: flex;
+    flex-direction: column;
+    padding-left: 8%;
+    width: 50vw;
+  }
+
+  .second-grid1 h2 {
+    font-size: 40px;
+  }
+
+  .text4 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    background: #ebebff;
+    border-radius: 8px;
+    color: #6070ff;
+    gap: px;
+    letter-spacing: 0.03em;
+  }
+
+  .page-4 {
+    display: none;
+  }
+
+  .page-4-desktop {
+    display: grid;
+    grid-template-columns: 40% 40%;
+    grid-template-rows: 1fr;
+    width: 80vw;
+    margin-top: 40%;
+    gap: 124px;
+    justify-items: stretch;
+    padding-left: 10%;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+
+  .page-4-desktop .bullets1,
+  h3 {
+    font-size: 15px;
+    line-height: 24px;
+    font-weight: 500;
+  }
+
+  .page-2-desktop .bullets1,
+  h3 {
+    font-size: 15px;
+    line-height: 24px;
+    font-weight: 500;
+  }
+
+  .page-3-desktop .bullets1,
+  h3 {
+    font-size: 15px;
+    line-height: 24px;
+    font-weight: 500;
+  }
+
+  .page-5-desktop .bullets1,
+  h3 {
+    font-size: 15px;
+    line-height: 24px;
+    font-weight: 500;
+  }
+
+  .page-5 {
+    display: none;
+  }
+
+  .page-5-desktop {
+    display: grid;
+    grid-template-columns: 40% 40%;
+    grid-template-rows: 1fr;
+    width: 80vw;
+    margin-top: 40%;
+    gap: 150px;
+    justify-items: stretch;
+    padding-left: 10%;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+
+  .headline-1 {
+    margin-top: 200px;
+  }
+
+  form {
+    grid-template-columns: 100vw;
+    background-image: url(contact-desktop.png);
+    background-position-x: 100px;
+    background-size: 85%;
+    background-position-y: 100px;
+    margin-top: 130px;
+  }
+
+  .contact {
+    display: grid;
+    grid-template-columns: 100%;
+  }
+
+  .contact h4 {
+    margin-top: 120px;
+  }
+
+  .contact p {
+    width: 550px;
+    text-align: center;
+  }
+
+  .input,
+  .text-area {
+    width: 450px;
+  }
+
+  .submit {
+    margin-right: 10px;
+  }
+
+  .move-last {
+    display: flex;
+    flex-direction: column;
+    margin-top: 30px;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  .page-5-desktop p {
+    width: 70%;
+  }
+
+  .container-last {
+    padding-left: 20%;
+  }
+
+  .down {
+    margin-left: 310px;
+  }
+
+  .but1 {
+    margin-top: 20px;
+  }
+
+  .right {
+    margin-left: 300px;
+  }
+
+  .right1 {
+    margin-left: 375px;
+  }
+}
+
+@media only screen and (min-width: 1440px) {
+  .but1 {
+    margin-top: -200px;
+  }
```

Comment:
- [ ] Please remove the gap and make it like the figma template. after fixing this your card will look much more better.
## your design👇
![getbtn](https://user-images.githubusercontent.com/66992901/193366248-0d7eaf50-267a-484b-962a-7eb2dbfb42ed.PNG)

## Figma design 👇
![getbtnFigma](https://user-images.githubusercontent.com/66992901/193366276-e11dcf88-f361-4c1d-93ed-ebed5283a01a.PNG)